contracts/rateproducer/README.md

```diff
@@ -1,28 +1,75 @@
+## Compilation
+This contract is build with eden `clsdk` compiler, to have the local environment configurated to compile this contract is needed to [take a look at this link](https://github.com/eoscommunity/demo-clsdk) to have it clear or just follow next steps to go forward.
```

Comment:
This contract is **_built_** with

Reply:
configurated is not a word , try `configured` also please run a spell check

Reply:
replace : to have it clear or just follow next steps to go forward.

with: to learn more or follow these steps to  continue forward